recipe/meta.yaml

```diff
@@ -23,14 +23,15 @@ requirements:
     - wheel
     - setuptools
   run:
-    - python >=3.6
+    # The package is not supported for the versions 3.0 to 3.3 of python
+    # https://github.com/jgm/pandocfilters/blob/master/setup.py#L19
+    - python !=3.0,!=3.1,!=3.2,!=3.3
 
 test:
   imports:
     - pandocfilters
   requires:
     - pip
-    - python <3.10
```

Comment:
If we use noarch, it's better to use `- python <3.10` in `test/requires` as for now we aren't building on python 3.10

Reply:
> If we use noarch, it's better to use `- python <3.10` in `test/requires` as for now we aren't building on python 3.10

The restrictions for python 3.10 were added
https://github.com/AnacondaRecipes/pandocfilters-feedstock/blob/update_to_1.5.0/recipe/meta.yaml#L35